### PR TITLE
Remove forbidden `@author` tag from `Magento_Review`

### LINKED ISSUE #37277: [Issue] Remove forbidden `@author` tag from `Magento_Review`
This issue is automatically created based on existing pull request: magento/magento2#36977: Remove forbidden `@author` tag from `Magento_Review`

---------
### Description
According to https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#documentation-space, the `@author` tag is not permitted in Magento. This pull request removes this tag from the `Magento_Review` module. Given there are so many instances of this tag, I've opened a small pull request to get the process started. I expect that the linter will force me to fix several other coding standards violations on the way, so having a smaller pull request means that task is easier to manage. I plan to open more pull requests to tackle the other instances of this tag.

See also https://github.com/magento/magento-coding-standard/pull/382 and https://github.com/magento/magento-coding-standard/issues/167

See also https://github.com/magento/magento2/pull/36304, https://github.com/magento/magento2/pull/36976

### Manual testing scenarios
There are not code changes in this pull request. This pull request only removes forbidden comments.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### PR DESCRIPTION
**Pull request by fredden**
### Description
According to https://devdocs.magento.com/guides/v2.4/coding-standards/docblock-standard-general.html#documentation-space, the `@author` tag is not permitted in Magento. This pull request removes this tag from the `Magento_Review` module. Given there are so many instances of this tag, I've opened a small pull request to get the process started. I expect that the linter will force me to fix several other coding standards violations on the way, so having a smaller pull request means that task is easier to manage. I plan to open more pull requests to tackle the other instances of this tag.

See also https://github.com/magento/magento-coding-standard/pull/382 and https://github.com/magento/magento-coding-standard/issues/167

See also https://github.com/magento/magento2/pull/36304, https://github.com/magento/magento2/pull/36976

### Manual testing scenarios
There are not code changes in this pull request. This pull request only removes forbidden comments.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All new or changed code is covered with unit/integration tests (if applicable)
 - [x] README.md files for modified modules are updated and included in the pull request if any [README.md predefined sections](https://github.com/magento/devdocs/wiki/Magento-module-README.md) require an update
 - [x] All automated tests passed successfully (all builds are green)

### Resolved issues:
1. [x] resolves magento/magento2#37277: Remove forbidden `@author` tag from `Magento_Review`

### CODE REVIEW

**Comment by fredden** [comment]
@magento run all tests

**Comment by fredden** [comment]
@magento run Static Tests

**Comment by fredden** [comment]
@magento run Static Tests

**Comment by fredden** [comment]
@magento run all tests

File: app/code/Magento/Review/Model/ResourceModel/Rating/Collection.php [hunk 1]
```diff
@@ -314,7 +311,9 @@ protected function _addStoreData()
         if (is_array($data) && count($data) > 0) {
             foreach ($data as $row) {
                 $item = $this->getItemById($row['rating_id']);
-                $item->setStores(array_merge($item->getStores(), [$row['store_id']]));
+                $stores = $item->getStores();
```
Comment (leonhelmus) [inline app/code/Magento/Review/Model/ResourceModel/Rating/Collection.php:314]: @fredden This seems Like a good change maybe add it in the desciption of the pull request.

**Review by leonhelmus — APPROVED** [review]
Looks good to me!

**Comment by engcom-Lima** [comment]
:heavy_check_mark: QA Passed

**Preconditions:**
- Install fresh Magento **`2.4-develop`** and **`PHP 8.1`**

**Manual testing scenario:**

-  Verified author tag got removed from `Review` module.

**Before:** :heavy_multiplication_x: 

Screenshot 2023-03-21 at 2 57 10 PM


**After:** :heavy_check_mark:  
Screenshot 2023-03-21 at 3 03 48 PM

No addiitional manual test cases is required for the regression as this PR related to the removal of the @author tag in sales module.

Multiple builds are failing hence, moving this PR to **Extended Testing**

**Comment by engcom-Lima** [comment]
@magento create issue

**Comment by engcom-Delta** [comment]
@magento run all tests

**Comment by engcom-Delta** [comment]
@magento run all tests

**Comment by engcom-Delta** [comment]
@magento run  Functional Tests B2B,  Functional Tests CE,Functional Tests EE

**Comment by engcom-Delta** [comment]
@magento run Functional Tests B2B, Functional Tests CE,Functional Tests EE

**Comment by engcom-Delta** [comment]
Unit Test failure is not related to the PR changes
Functional Tests B2B and CE failures are different on last two run
Functional B2B : run 1
Screenshot b2b 2023-03-27 at 10 47 26 AM
run 2
Screenshot 2023-03-27 at 3 03 24 PM
Functional CE : run 1
Screenshot 2023-03-27 at 3 05 26 PM
run 2
Screenshot 2023-03-27 at 7 09 50 PM

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
@magento run Integration Tests,Functional Tests CE,Functional Tests EE,Functional Tests B2B

**Comment by engcom-Echo** [comment]
Functional Tests EE,Functional Tests CE,Functional Tests B2B failures are different on last two run.
Functional Tests EE
Run1
36977-EE

Run2
36977-ee2

Functional Tests CE
Run1
36977-CE

Run2
36977-ee2

Functional Tests B2B
Run1
36977-B2B

Run2
36977-b2b2

Hence moving it to `merge in progress`

**Pull request merged**